src/main/java/gift/domain/member/Member.java

```diff
@@ -0,0 +1,58 @@
+package gift.domain.member;
+
+public class Member {
+    private Long id;
+    private String email;
+    private String name;
+    private String password;
+    private int role;
+
+    public Member(String email, String name, String password, int role) {
+        this.email = email;
+        this.name = name;
+        this.password = password;
+        this.role = role;
+    }
+
+    public Member(Long id, String email, String name, String password, int role) {
+        this.id = id;
+        this.email = email;
+        this.name = name;
+        this.password = password;
+        this.role = role;
+    }
+
+
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public int getRole() {
+        return role;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public boolean isMatch(String input){
+        return input.equals(password);
```

Comment:
이 메서드만 봤을 때 input이 무엇인지 알 수 없는 상태이니 password로 이름을 바꾸면 어떨까요?

추가로, null이라면 NullPointerException이 발생하기에 equals 비교시에 null이 아닌 값이 선행되어야 합니다.  
Member 생성시 password가 유효한지 null 체크를 해두면 더 좋습니다.
```
    public Member(String email, String name, String password, int role) {
        validatePassword(password);
        // ... 생략
        this.password = password;
    }
```
```suggestion
        return this.password.equals(password);
```